src/Microsoft.Health.Fhir.TemplateManagement/Microsoft.Health.Fhir.TemplateManagement.csproj

```diff
@@ -52,36 +52,42 @@
       <Link>JsonDefaultTemplates.tar.gz</Link>
     </EmbeddedResource>
   </ItemGroup>
-	
+
   <ItemGroup>
     <None Update="oras.exe">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
-    
+
   <ItemGroup>
     <None Update="oras-OSX">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
-  
+
+  <ItemGroup>
+    <None Include="$(BinFolder)oras_$(OrasVersion)_darwin_amd64.tar.gz">
```

Comment:
We can rename the package name to a fixed name so that others who use the tool don't need to change the filename when the version or filename is changed.

Reply:
Thanks, the tar.gz file for mac will be fixed as "oras_osx_amd64.tar.gz"